docs/host/host.md

```diff
@@ -279,6 +279,21 @@ The replica data would be rebuilt to another disk automatically to keep the high
 4. Select **Save** to remove the disk.
 
 
+## Topology spread constraints
+
+_Available as of v1.1.0_
+
+Topology spread constraints rely on node labels to identify the topology domain(s) that each node is in.
+The labels can be set up through the Harvester UI on the host page:
```

Comment:
```suggestion
## Topology Spread Constraints

[Node labels](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#node-labels) are used to identify the topology domains that each node is in. You can configure labels such as [`topology.kubernetes.io/zone`](https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone) on the Harvester UI.
```

Reply:
I slightly modified the sentence that you lifted from the Kubernetes doc (to avoid issues) and added links to relevant topics.